templates/bareos-dir/storage/storage.conf.j2

```diff
@@ -1,7 +1,7 @@
 # {{ ansible_managed }}
 Storage {
   Name = "{{ item.name }}"
-  Address = "{{ bareos_dir_ip_ib | default(bareos_dir_ip_eth) }}"
+  Address = "{{ item.bareos_dir_ip }}"
```

Comment:
I don't understand this change. Could you explain the why?

Reply:
We need to specifiy the IP for each storage depending on the client. In some cases we need IB, in others eth.
As is, all the storages will be configured with IB or all the storages will be configured with eth, whch is not what we want.